imjoy/engine.py

```diff
@@ -2,14 +2,21 @@
 import logging
 import sys
 
-from imjoy.helper import setup_logging
 from imjoy.options import parse_cmd_line
 from imjoy.env import bootstrap, prepare_env
 from imjoy.connection import create_connection_manager
 from imjoy.runners import setup_runners
 from imjoy.services import setup_services
 
 
+def setup_logging(opt, logger):
```

Comment:
We could keep it here until we create the new logging set up function that will be a hit bigger.

Reply:
Or move it to utils.py ?

Reply:
Maybe. If we add custom logging handlers it might become more than a util I'm thinking. Does it deserve its own module? For now I think we can keep it here.